JeMPI_Apps/JeMPI_UI/src/utils/formatters.ts

```diff
@@ -37,3 +37,8 @@ export const formatName = (value: string) => {
   const fullName = value.split(',')
   return `${fullName[0] ? fullName[0] : ''} ${fullName[1] ? fullName[1] : ''}`
 }
+
+export const truncateString = (value: string, maxLength: number) => {
+  if (value.length > maxLength) return `${value.substring(0, maxLength)}...`
```

Comment:
Should we bit have some type schecking here. Someone could accidently `null` which would result in a error....and maybe the name `truncateString` is a bit misleading as you a truncating and then formatting (with the 3 dots). Maybe `summarizeString` could be a better name?

Reply:
the name truncate is more like the technical word for ellipsis the string in Web dev 
more sources : 
https://www.freecodecamp.org/news/how-to-truncate-text-with-css-javascript/
for null handling, the function does not accept any types just string (type checking already exists) so any variable that could be null will throw an error (dev phase), for the use case there is no OS accepting a file without a name.

Reply:
Hi @issambaccouch. Thanks for the feedback. A string can be `null` in typescrript, it's more to do with how you set up your ts compilier settings.Regardless, typescript is transpilied to standard js,, which means if someone calls the compiled version the error would occur (i.e suppose someone adds a `.js` file later to the repository and call this method. No typechecking will happen on their file, yet it will cause a runtime error). p.s the other reason I am pointing it out is becuase you have added the function to the `utils` folder (which acts like a global/common folder). i.e it should usuable by all files. If it was more localised (or encupsulated in a class) I think I would understand it more.

re:- naming. Thanks for the link. Maybe call is `truncateStringEllipsis` then? Again, people using this "global/common" function might assume it just does a strict truncate, not knowing it add ellipsis to the end

Reply:
for a better approach using CSS is better than using javascript for truncating the text, please check this commit 
https://github.com/jembi/JeMPI/pull/126/commits/54a1cc053c59b4cbcdceed371f5a3b7cd8944729